### PR TITLE
Added user edit page, avatar to user and tests to it

### PR DESCRIPTION
**Pull request by sfrunza**
Hey i have the nav bar ready with some updates, please pull it

### CODE REVIEW

File: app/javascript/packs/application.js [hunk 1]
```diff
@@ -7,15 +7,4 @@
...
-import React from 'react'
-import ReactDOM from 'react-dom'
-import PropTypes from 'prop-types'
-import App from '../App'
-
-ReactDOM.render(
-  <App/>,
-  document.getElementById('app')
-);
+console.log('Hello World from Webpacker')
```
Comment (bkincart) [inline app/javascript/packs/application.js]: I don't think you want to delete the work from yesterday here, correct? Please be sure to run the following to fix this and re-push:
```
git checkout master
git pull origin master
git checkout nav-bar
git merge master
```

File: app/views/devise/registrations/edit.html.erb [hunk 2]
```diff
@@ -31,13 +31,37 @@
...
+    <%= f.hidden_field :avatar_cache %>
+    <% end %>
+    <label>
+      <%= f.check_box :remove_avatar %>
+      Remove avatar
+    </label>
+  </p>
+  </div>
+  <% end%>
+
```
Comment (bkincart) [inline app/views/devise/registrations/edit.html.erb]: Nice conditional view!

File: spec/features/user_edit_profile_spec.rb [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+    click_button 'Log in'
+
+    expect(page).to have_content('Signed in successfully')
+    expect(page).to have_content('Sign Out')
+
+    visit edit_user_registration_path
+
+    fill_in 'user[current_password]', with: user.password
+    click_button 'Update'
```
Comment (bkincart) [inline spec/features/user_edit_profile_spec.rb]: Could we add an expectation that the field is there to upload an avatar?

**Review by bkincart — CHANGES_REQUESTED** [review]

**Comment by sfrunza** [comment]
hey my tests are passing! please take a look

File: app/javascript/packs/application.js [hunk 4]
```diff
@@ -7,4 +7,15 @@
 // To reference this file, add <%= javascript_pack_tag 'application' %> to the appropriate
 // layout file, like app/views/layouts/application.html.erb
 
-console.log('Hello World from Webpacker')
+console.log('You\'re doing great!')
```
Comment (dpickett) [inline app/javascript/packs/application.js]: we should eliminate any `console.log` or `binding.pry` statements prior to pull requesting.

**Review by dpickett — APPROVED** [review]

**Pull request merged**